src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -54,7 +54,29 @@
 
 
 class FlowExecutor:
-    """This class is used to execute a single flow for different inputs."""
+    """This class is used to execute a single flow for different inputs.
+
+    :param flow: The flow to be executed.
+    :type flow: Flow
```

Comment:
use `~promptflow.xxx.Flow` to point to the class, it will be rendered to an icon which points to the class doc.